src/otx/algo/detection/utils/yolov7_v9_utils.py

```diff
@@ -0,0 +1,256 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Utils for YOLOv7 and v9.
```

Comment:
Can't we merge this file with general detection utils? The naming overall looks a bit strange and I think we can combine these bbox related utils with other "utils"

Reply:
Done at d106c7c